OpenEdX/Router.swift

```diff
@@ -18,24 +18,29 @@ import Dashboard
 import Profile
 import WhatsNew
 import Combine
- 
+
+// swiftlint:disable file_length type_body_length
 public class Router: AuthorizationRouter,
                      WhatsNewRouter,
                      DiscoveryRouter,
                      ProfileRouter,
                      DashboardRouter,
                      CourseRouter,
                      DiscussionRouter {
-    
+
     public var container: Container
-    
+
     private let navigationController: UINavigationController
-    
+
+    func getNavigationController() -> UINavigationController {
+        navigationController
+    }
```

Comment:
How about changing the access modifier of `navigationController` to public instead of defining a method. It's a read-only property so its not possible to change it from outside. Thoughts?

Reply:
I think we don't need have public navigationController, I use only for deep link